examples/slo/main.tf

```diff
@@ -112,3 +112,36 @@ resource "chronosphere_slo" "slo_with_signal_grouping_labels" {
     }
   }
 }
+
+resource "chronosphere_slo" "slo_with_filters" {
+  name                   = "SLO with filters"
+  collection_id          = chronosphere_collection.c.id
+  notification_policy_id = chronosphere_notification_policy.np.id
+
+  definition {
+    objective = 99.95
+    reporting_windows {
+      duration = "28d"
+    }
+  }
+
+  sli {
+    custom_indicator {
+      bad_query_template   = "sum(rate(http_request_duration_seconds_count{error=\"true\", {{ .AdditionalFilters }} }[{{ .Window }}]))"
```

Comment:
optional: consider using a multiline heredoc
https://developer.hashicorp.com/terraform/language/expressions/strings

that way don't have to escape the quotes, and can make the expression more readable (especially with all the braces+brackets, my head hurts a little seeing `}}}[{{`)